### ISSUE TITLE
Ajouter un service permettant de gérer le résultat

### ISSUE THREAD

**Issue by GuenotF** [Nicolas-Hermet/ProGGG#6]
### Ajouter un service permettant de gérer le résultat de la partie.
Ce service doit pouvoir :
- Augmenter le score
- Réinitialiser le score
- Permettre d'accéder au score
- Lister les chansons ratées
- Ajouter une chanson si elle est ratée
- Réinitialiser la liste des chanson ratées
- ... et plus si affinités.

**Comment by Nicolas-Hermet** [comment]
> - Ajouter une chanson si elle est ratée.

Comment ça ?

> - Réinitialiser la liste des chanson ratées

Comment ça ?

Tu me détaillerais le use case que tu as en tête pour ces deux points ?

Sinon deux remarques à deux balles : mais je veux bien qu'on parle anglais :grin: et je pense qu'il faut essayer de limiter les scopes des issues. Donc ici pas de "et plus si affinité". Au pire si on doit ou veut rajouter un truc, on écrira une issue de plus.
Ça te vas ? Ou tu préfères qu'on fasse autrement ?

**Comment by GuenotF** [comment]
Je me suis basé sur le jeu déjà existant et l'idée c'est de pouvoir lister les chansons ratées par le joueur à la fin de la partie. Donc t'aurais le score (nombre de chansons réussies et le nom des chansons ratées. C'est une idée mais on est pas obligé de mettre ça en place pour l'instant. 

Si tu veux on parle anglais pas de soucis pour moi donc toutes les PR, issues et les commentaires qu'on fait sont en anglais à partir de maintenant ?

**Comment by Nicolas-Hermet** [comment]
> Je me suis basé sur le jeu déjà existant et l'idée c'est de pouvoir lister les chansons ratées par le joueur à la fin de la partie. Donc t'aurais le score (nombre de chansons réussies et le nom des chansons ratées. C'est une idée mais on est pas obligé de mettre ça en place pour l'instant.
> 
> Si tu veux on parle anglais pas de soucis pour moi donc toutes les PR, issues et les commentaires qu'on fait sont en anglais à partir de maintenant ?

Yep, so indeed, we should switch to english right away. Otherwise how would this be open source right ?

Anyway, I asked these because I had a hard time understan your PR description without reading the code. No changes needed here but do you think you can add a bit more detailed on your next PR ?
Otherwise, and appart from tests, I think its great :+1:
